code/modules/flufftext/Dreaming.dm

```diff
@@ -5,12 +5,20 @@
 /mob/living/carbon/proc/dream()
 	set waitfor = FALSE
 	var/list/dream_fragments = list()
+	var/list/custom_dream_nouns = list()
 	var/fragment = ""
 
+	for(var/obj/item/bedsheet/sheet in loc)

```

Comment:
brain-toot:

loc is an instance not a list, so does byond understand this to mean `loc.contents`?

Reply:
for i in object defaults to for i in object.contents

Reply:
He's correct, that code is actually copied directly from the original version of the file before the dreams rework accidentally overwrote this vital functionality.

Reply:
Alright thanks, you cured my brain-toot.